src/test/resources/features/PresentationNotificationFE/IE13PresentationNotificationFE.feature

```diff
@@ -93,31 +88,40 @@ Feature: Manage transit movements frontend tests for Departures
     And I enter identification number GB1234567 on 'What is the identification number for this?' page
     And I select United Kingdom on the active border mode 'What country is this vehicle registered to?' page
     And I select Belfast EPU (XI000142) on 'Where is the customs office at the border?' page
-    And I choose radio Yes on the presentation 'Do you want to add a conveyance reference number?' page
-    And I enter conveyance reference number GB123456123456 on the 'What is the conveyance reference number?' page
-    And I choose radio Yes on the presentation 'You have added 1 border means of transport' page
-    And I choose radio IATA flight number on the presentation inland mode 'Which identification do you want to use for this vehicle?' page
-    And I enter identification number GB1234567 on 'What is the identification number for this?' page
-    And I select United Kingdom on the active border mode 'What country is this vehicle registered to?' page
-    And I select Belfast EPU (XI000142) on 'Where is the customs office at the border?' page
     And I choose radio No on the presentation 'Do you want to add a conveyance reference number?' page
-    And I click the Remove link  on the 'You have added 2 border means of transport' page
-    And I click radio Yes on the 'Are you sure you want to remove this border means of transport?' page
     And I choose radio Yes on the presentation 'You have added 1 border means of transport' page
 
-  Scenario: 04 Trader completes a Simplified departure declaration that was previously pre-lodged - includes Container Indicator, Limit Date, Place of Loading and Auth Type
-    When I submit an IE015 Simplified Prelodged Departure Declaration Full
+      Scenario: 04 Trader completes a Normal departure declaration that was previously pre-lodged - Answer no to optional questions
```

Comment:
BEing pedantic sorry :) Do we want to change the scenario name to follow the pattern we started in the IE15?